src/OpenAI.csproj

```diff
@@ -28,7 +28,12 @@
    
     <!-- Embed source files that are not tracked by the source control manager in the PDB -->
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
-</PropertyGroup>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <!-- Allow use of unsafe code, for System.Net.ServerSentEvents polyfill on netstandard2.0 -->
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Could you add a comment and file an issue to remove this once we switch to using the oob package?

Reply:
Yup.

Reply:
Done. Opened https://github.com/openai/openai-dotnet/issues/41 and updated the csproj with a comment.